modules/dqt/jsx/react.tabs.js

```diff
@@ -184,6 +184,19 @@ class ViewDataTabPane extends Component {
     this.downloadDataCSV = this.downloadDataCSV.bind(this);
   }
 
+  /**
+   * Called by React when the component has updated.
+   * @param {object} prevProps - Previous component properties
+   * @param {object} prevState - Previous component state
+   */
+  componentDidUpdate(prevProps, prevState) {
```

Comment:
@ridz1208 this change is part of the solution for the "Fixes page error when trying to filter on select elements" bug. this.state.sessionsLoaded will be set to true once sessions have been retrieved and so if sessions are emptied then the run query button doesn't get reset depending on if fields are selected.